Uplift/Views/GymDetail/GymDetailTimeInfoView.swift

```diff
@@ -161,16 +163,20 @@ class GymDetailTimeInfoView: UIView {
         self.emptyStateView = emptyStateView
     }
 
-    func getTimesHeight(for hours: [FacilityHoursRange]) -> CGFloat {
-        let emptyStateHeight: CGFloat = 163
-        let hoursCount = CGFloat(hours.count)
+    static func getTimesHeight(for dailyHours: [DailyFacilityHoursRanges]) -> CGFloat {
+        var timesHeight = emptyStateHeight
+        dailyHours.forEach { dailyHour in
+            timesHeight = max(timesHeight, getTimesHeight(for: dailyHour.timeRanges))
+        }
+        return timesHeight
```

Comment:
you should be able to simplify this using `max(by: )` function of arrays (https://developer.apple.com/documentation/swift/array/2294243-max)